### DISCUSSION TITLE
仅有ipv6公网ip，公网ip域名解析成功访问，添加至alist出错？

### DISCUSSION THREAD

**Discussion by mangic** [AlistGo/alist#933] [Q&A]
![image](https://user-images.githubusercontent.com/33594145/163088653-7c20e835-9bf6-43d2-81dd-9de73a6f3a18.png)

**是否与alist不支持添加仅有ipv6地址有关？**

**Comment by xhofe** [comment]
无关。

**Reply by mangic** [reply]
> 无关。
已解决，和自身服务器有关，自身服务器没有开启ipv6，自然也没有开放ipv6的端口访问
